runner.go

```diff
@@ -32,13 +33,22 @@ func init() {
 		os.Exit(2)
 	}
 	log.SetOutput(f)
+
+	config := &Config{
```

Comment:
can this be configurable via flag or something? just don't want it on by default